src/Element.php

```diff
@@ -85,7 +85,7 @@ public function init()
      * Set value for name
      *
      * @param  string $name
-     * @return Element|ElementInterface
+     * @return static
```

Comment:
For the fluent interface there is a special return type:
```suggestion
     * @return $this
```